src/prog_models/prognostics_model.py

```diff
@@ -405,6 +405,51 @@ def next_state(self, x, u, dt) -> dict:
         dx = self.dx(x, u)
         return {key: x[key] + dx[key]*dt for key in dx.keys()}
 
+    def __next_state(self, x, u, dt) -> dict:
+        """
+        State transition equation: Calculate next state and add noise
```

Comment:
Could you add a comment here explaining the difference between __next_state and next_state and how next_state is meant to be overwritten. 

Same with output, below